website/versioned_docs/version-1.2/api/spec.md

```diff
@@ -11,10 +11,9 @@ A Desktop Agent is a desktop component (or aggregate of components) that serves
 
 Examples of Desktop Agents include:
 
-- Autobahn
 - Cosaic's Finsemble
+- Glue42
 - OpenFin
-- Refinitiv Eikon
```

Comment:
Didn't Refinitiv talk about work they were doing on FDC3 at a meeting a couple of months ago? I would suggest leaving this in.

Reply:
@ggeorgievx @kriswest same with Autobahn actually - let's not use this change to remove others, let's just add Glue42.